frontend/package.json

```diff
@@ -11,17 +11,18 @@
     "codecov": "codecov",
     "lint": "ng lint > tslint-result.txt",
     "e2e": "ng e2e",
-    "postinstall": "echo 'Replacing moment.js by empty file since it is optional and not used'; echo '' > node_modules/moment/moment.js"
+    "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points && echo 'Replacing moment.js by empty file since it is optional and not used'; echo '' > node_modules/moment/moment.js"
```

Comment:
Yes, but this has been reverted in latest version, so the final version won't add it, because it caused some issues. If it's not done in a postinstall step, then the CLI runs on the first ng serve/build automatically (but a slower version, this one is parallelized, whereas the CLI one is not). So we can keep it, as it is a bit faster.

Reply:
Even if it's a bit slower, I'd prefer to do it the standard way.